check

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash -ex
+bundle exec rake lint test-zonefile
+bundle exec parallel_rspec spec/classes
```

Comment:
The thing about parallel_rspec is that it's not the same as running the `spec` task in Rake.

In order for this to work properly, you would have to make sure that you had run `bundle exec rake spec_prep` at least once before hand to prepare the `spec/fixtures` directory